src/test/java/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCMStepDescriptorTest.java

```diff
@@ -45,8 +48,8 @@ public void testDoCheckServerId() {
 
     @Test
     public void testDoFillCredentialsIdItems() {
-        descriptor.doFillCredentialsIdItems("myBaseUrl", "myCredentialsId");
-        verify(formFill).doFillCredentialsIdItems("myBaseUrl", "myCredentialsId");
+        descriptor.doFillCredentialsIdItems(item, "myBaseUrl", "myCredentialsId");
```

Comment:
I think here (and other POST methods we're testing) we should be mocking item privileges to ensure they're insufficient privileges are being rejected. That's a blind spot in the code base in general though, and I don't think it's needed to merge this PR.